src/packages/cdk/src/examples/lambda.ts

```diff
@@ -68,6 +68,7 @@ export const graphweaverApp = new GraphweaverApp(rootStack, 'TestGraphweaverDock
 	},
 	lambda: {
 		packageName: '@exogee/graphweaver',
+		functionName: 'Graphweaver API',
```

Comment:
Given that the space isn't a valid character, we should probably do this for the example:

```suggestion
		functionName: 'Graphweaver_API',
```